.github/workflows/ci.yml

```
@@ -70,14 +72,18 @@ jobs:
 
     strategy:
       matrix:
-        angular-version: [9.0.x, 9.1.x, 10.0.x, 10.1.x, 10.2.x, 11.0.x, 11.1.x, 11.2.x]
+        node-version: [12.x]
+        angular-version: [9.0.x, 9.1.x, 10.0.x, 10.1.x, 10.2.x, 11.0.x, 11.1.x, 11.2.x, 12.0.x]
+        include:
+          - node-version: 14.x
+            angular-version: 12.0.x
```

suggestion: Use this format for consistency:
```suggestion
          - angular-version: 12.0.x
            node-version: 14.x
```